Firmware/tmc2130.cpp

```diff
@@ -963,7 +963,7 @@ void tmc2130_set_wave(uint8_t axis, uint8_t amp, uint8_t fac1000)
 		else
 			reg >>= 1;
 //		printf("%3d\t%3d\t%2d\t%2d\t%2d\t%2d    %08x\n", i, vA, dA, b, w[s], s, reg);
-	}
+	} while (i++ != 255);
```

Comment:
@gudnimg please double check if it behaves correctly now

Reply:
The old code would loop indefinitely since uint8_t is always smaller than 256.

Reply:
Ah I see, can't believe I missed that.